website/docs/r/automation_actions_action.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_automation_actions_action"
+sidebar_current: "docs-pagerduty-resource-automation-actions-action"
+description: |-
+  Creates and manages an Automation Actions action in PagerDuty.
+---
+
+# pagerduty\_automation\_actions\_action
+
+An Automation Actions [action](https://developer.pagerduty.com/api-reference/d64584a4371d3-create-an-automation-action) invokes jobs and workflows that are staged in Runbook Automation or Process Automation. It may also execute a command line script run by a Process Automation runner installed in your infrastructure.
+
+## Example Usage
+
+```hcl
+resource "pagerduty_automation_actions_action" "example" {
+  name = "PA Action created via TF"
+  description = "Description of the PA Action created via TF"
+  action_type = "process_automation"
+  action_data_reference {
+    process_automation_job_id = "P123456"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+  * `name` - (Required) The name of the action.
+  * `description` - (Required) The description of the action.
+  * `action_type` - (Required) The type of the action. The only allowed values are `process_automation` and `script`. 
+  * `action_data_reference` - (Required) Action Data block. Action Data is documented below.
+  * `runner_id` - (Optional) The Process Automation Actions runner to associate the action with. 
+  * `action_classification` - (Optional) The category of the action. The only allowed values are `diagnostic` and `remediation`. 
+
+Action Data (`action_data_reference`) supports the following:
+
+  * `process_automation_job_id` - (Required for `process_automation` action_type) The ID of the Process Automation job to execute.
+  * `process_automation_job_arguments` - (Optional) The arguments to pass to the Process Automation job execution.
+  * `script` - (Required for `script` action_type) Body of the script to be executed on the Runner.
```

Comment:
@mrdubr In this field I consider is needed the characters limit accepted by the API.